src/main.js

```diff
@@ -177,6 +184,9 @@ function TimekitBooking() {
       // Render available timeslots in FullCalendar
       if(slots.length > 0) renderCalendarEvents(slots);
 
+      // Render test ribbon if enabled 
+      if (response.headers['timekit-testmode']) renderTestModeRibbon();
```

Comment:
Shouldn't this be in PascalCase too? I'd be surprised if tests pass as object keys are case sensitive. Maybe Axios converts them to lowercase automatically?